conf/docker-aio/readme.md

```diff
@@ -27,7 +27,7 @@ Also usable for normal development and system evaluation; not intended for produ
 #### Setup
 
 - `cd conf/docker-aio`, and run `./1prep.sh` to copy files for integration test data into docker build context; `1prep.sh` will also build the war file and installation zip file
-- build the docker image: `docker build -t dv0 -f c7.dockerfile .`
+- build the docker image: `docker build -t dv0 -f c8.dockerfile .`
```

Comment:
updating `prep_it.bash` with the corresponding change is probably also helpful.

Reply:
Thanks, fixed in 7f90961.